### PR TITLE
Remove all svg icon.

### LINKED ISSUE #4157: Replace icons in `@yamada-ui/close-button` package with `LucideIcon`
### Description

Replace icons in `@yamada-ui/close-button` package with `LucideIcon`.

### Problem Statement/Justification

Replace the icons from `@yamada-ui/icon` with those from `@yamada-ui/lucide`.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

- Use a similar icon from `@yamada-ui/lucide`.
- Maintain the size, style, and overall design to avoid significant changes.
- Replace and remove any custom SVG icons being used.
- Check the appearance to ensure it still looks good.
- Check out from `v2` and merge into `v2`.

### PR DESCRIPTION
**Pull request by illionillion**
Related #3814  #3535 #4161 #4167 #3533
Closes #4157 #4152 #4160 #4159

## Description

Replaced all icons made with SVG in /icon/**.

### CODE REVIEW

File: packages/react/src/components/autocomplete/autocomplete-option.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import type { CSSUIObject, FC } from "../../core"
 import type { UseAutocompleteOptionProps } from "./use-autocomplete-option"
 import { ui } from "../../core"
 import { cx } from "../../utils"
-import { CheckIcon } from "../icon/icons"
+import { CheckIcon } from "../icon/"
```
Comment (Itsuki54) [inline packages/react/src/components/autocomplete/autocomplete-option.tsx:6]: Shouldn't the / at the end be removed?

File: packages/react/src/components/autocomplete/autocomplete-icon.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { cloneElement, useRef } from "react"
 import { ui } from "../../core"
 import { useClickable } from "../../hooks/use-clickable"
 import { cx, getValidChildren, isValidElement } from "../../utils"
-import { ChevronDownIcon, XIcon } from "../icon/icons"
+import { ChevronDownIcon, XIcon } from "../icon/"
```
Comment (Itsuki54) [inline packages/react/src/components/autocomplete/autocomplete-icon.tsx:7]: Shouldn't the / at the end be removed?

**Review by Itsuki54 — COMMENTED** [review]
I'd like to ask you to confirm this, as it may be a snippet of information.

File: packages/react/src/components/autocomplete/autocomplete-option.tsx (hunk 1 above)
Comment (illionillion) [reply packages/react/src/components/autocomplete/autocomplete-option.tsx:6]: You're right. I'll turn it off.

**Review by Itsuki54 — APPROVED** [review]
thank you fix LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**